hassio/src/addon-view/info/hassio-addon-info.ts

```diff
@@ -404,6 +404,7 @@ class HassioAddonInfo extends LitElement {
                 ? html`
                     <img
                       class="logo"
+                      alt=""
```

Comment:
The addon name is rendered just prior to this image, therefore I believe the image to be decorative.